README.md

```diff
@@ -735,28 +737,43 @@ IConfigurationSection variantConfiguration = variant.Configuration;
 // Do something with the resulting variant and its configuration
 ```
 
-### Setting a Variant's Configuration
+### Defining a Feature's Variants

```

Comment:
I think "Defining Variants" would be better since the above section is called "Getting Variants"

Reply:
I like the suggestion.

Reply:
Same with `Allocating Variants`